test/CDB_QuotaTest_expect

```diff
@@ -9,7 +9,7 @@ INSERT 0 2048
 INSERT 0 2048
 INSERT 0 2048
 2
-ERROR:  Quota exceeded by 567.998046875KB
+ERROR:  Quota exceeded by 519.998046875KB
 0
```

Comment:
I feel like I've seen this swing back and forth a bit... I have a feeling there's a platform dependence in the code somewhere.